hello.go

```diff
@@ -20,7 +20,6 @@ func main() {
 	defer client.Close()
 
 	// Sets the text to analyze.
-	//text := "Hello, world!"
 	text := "It's frustrating!"
```

Comment:
While you are here, can you convert the sentence to something positive?

Reply:
Done.